### ISSUE TITLE
Build fails because of dependency on broken nan 2.7.0

### ISSUE THREAD

**Issue by Xiretza** [meetfranz/franz#1027]
### Expected Behavior
Franz builds correctly.

### Current Behavior
Due to the dependency on broken `nan` version 2.7.0 in [yarn.lock](https://github.com/meetfranz/franz/blob/master/yarn.lock#L4371), the build fails.

This issue is documented in nodejs/nan#793 and was fixed back in November with [v2.8.0](https://github.com/nodejs/nan/releases/tag/v2.8.0).

For completeness and easier searching, I'll repost the error here:
```
In file included from ../../nan/nan.h:192,
                 from ../src/keyboard-layout-manager.h:4,
                 from ../src/keyboard-layout-manager-linux.cc:1:
../../nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
../../nan/nan_maybe_43_inl.h:112:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
   return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
               ^~~~~~~~
```

### Possible Solution
Update the yarn dependencies to at least `nan` v2.8.0.

### Your Environment
* Franz Version used: 5.0.0-beta.18
* Operating System and version: latest Arch Linux

**Comment by extremeshok** [comment]
+1 Broken on Manjaro / Arch running node 10.6

**Comment by polifolia** [comment]
+1

**Comment by Xiretza** [comment]
This was indeed solved in dd99f068d16ba34c8e53e3f87d8a3410378ddb64.
